CHANGELOG.md

```diff
@@ -1,12 +1,44 @@
 # Chain Core changelog
 
-Changelogs are maintained for each sub-package
-
-- [cored](./cmd/cored/CHANGELOG.md) (Chain Core server)
-- Installers
-  - [Mac](./installer/mac/CHANGELOG.md)
-  - [Windows](./installer/windows/CHANGELOG.md)
-- SDKs
-  - [Java](./sdk/java/CHANGELOG.md)
-  - [Node.js](./sdk/node/CHANGELOG.md)
-  - [Ruby](./sdk/ruby/CHANGELOG.md)
+This changelog covers updates to cored, the Chain Core server binary.
+
+For updates to subpackages, see below:
+
+- [Mac install](./installer/mac/CHANGELOG.md)
+- [Windows installer](./installer/windows/CHANGELOG.md)
+- [Java SDK](./sdk/java/CHANGELOG.md)
+- [Node.js SDK](./sdk/node/CHANGELOG.md)
+- [Ruby SDK](./sdk/ruby/CHANGELOG.md)
+
+## 1.1.0 (February 24, 2017)
+
+This release is a minor version update, and contains **new features** and **deprecations**. cored 1.1.0 is backward-compatible with 1.0.x SDKs, but we strongly recommend upgrading to 1.1.x SDKs as soon as possible. See SDK-level changes here:
+
+- [Java](./sdk/java/CHANGELOG.md)
```

Comment:
They might be more relevant if I had hardlinks into the specific version changes on those changelogs, but it's kinda an edge case. I'm down to just reducing this to "Please refer to the SDK changelogs for instructions on how to work with deprecations" or something to that effect.